test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,27 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() {
```

Comment:
We generally place test methods (those marked with Test annotation) together, while helper methods (such as assertNotUnique()) are placed at the bottom of the class file. Perhaps you can shift this method above so that helper methods can be found at the bottom of the class file.